import_precheck/README.md

```diff
@@ -0,0 +1,20 @@
+# Image Import Precheck Tool
+Precheck is run on your virtual machine before attempting to import it into
+Google Compute Engine (GCE). It attempts to identify compatibility issues that
+will either cause import to fail or will cause potentially unexpected behavior
+post-import. See our [image import documentation](../docs/image_import.md)
```

Comment:
Change this link to:

    [image import documentation](https://github.io/image_import.md)

Reply:
Done.